packages/financial-templates-lib/src/price-feed/CreatePriceFeed.js

```diff
@@ -107,28 +107,34 @@ async function createPriceFeed(logger, web3, networker, getTime, config) {
       config.priceFeedDecimals, // This defaults to 18 unless supplied by user
       uniswapBlockCache
     );
-  } else if (config.type === "defipulsetvl") {
-    const requiredFields = ["lookback", "minTimeBetweenUpdates", "defipulseApiKey"];
+  } else if (config.type === "defipulse") {
+    const requiredFields = ["lookback", "minTimeBetweenUpdates", "defipulseApiKey", "project"];
+    const VALID_PROJECTS = ["all", "SushiSwap", "Uniswap"];
 
     if (isMissingField(config, requiredFields, logger)) {
       return null;
     }
 
+    if (!VALID_PROJECTS.includes(config.project)) {
```

Comment:
do you think project should default to one of these, maybe `all`?

Reply:
also can we make this a case insensitive comparison

Reply:
I think we stay away from defaulting to all.  IMO the "all" synth may be hard to sustain without a large fee paid to LPs. I don't know many people who are shorting Defi in the long term.  Most likely we will find more users willing to short individual projects.

Also, it seems the case matters in the API request. "all" returns a response but "All" returns an error.